pages/en/contrib/code-contrib.md

```diff
@@ -80,6 +80,20 @@ All LoopBack projects follow the same convention for running tests. To run a pro
 npm test
 ```
 
+## Create example repositories
+
+If you create an example repository for any usage demo, the name of repository should follow the following convention:
+
+`loopback{major_version}-example-{example_name}`
+
+Usually the `{major_version}` is the lastest major version under development.
+If the example aims at a specific version of LoopBack, then use that number as the `{major_version}`.
+The `{example_name}` is a few words briefly describes the functionality of the example,
```

Comment:
can we reword to `a few words which briefly describe` here?

Reply:
done, thanks!